payments-core/src/main/java/com/stripe/android/payments/core/authentication/IntentAuthenticatorRegistry.kt

```diff
@@ -5,12 +5,10 @@ import com.stripe.android.model.StripeIntent
 /**
  * Registry to map [StripeIntent] to the corresponding [IntentAuthenticator] to handle its next_action
  */
-internal interface IntentAuthenticatorRegistry {
+internal fun interface IntentAuthenticatorRegistry {
 
     /**
      * Returns the correct [IntentAuthenticator] to handle the [StripeIntent].
      */
     fun getAuthenticator(stripeIntent: StripeIntent): IntentAuthenticator
-
-    // TODO(ccen): Add registration API
```

Comment:
all registration happens through multibinding, there will be no explicitly registration APIs exposed